apollo-execution-processor/src/main/kotlin/com/apollographql/execution/processor/definitions.kt

```diff
@@ -96,7 +96,7 @@ private class TypeDefinitionContext(
     while (declarationsToVisit.isNotEmpty()) {
       val declarationToVisit = declarationsToVisit.removeFirst()
       getOrResolve(declarationToVisit)
-    }
```

Comment:
Can revert this

Reply:
Indent fixes are very welcome 👍 . The project is moving fast, I don't think we want a separate commit for that.